spacy_llm/backends/rest/backend/openai.py

```diff
@@ -58,10 +59,10 @@ def credentials(self) -> Dict[str, str]:
         if api_org:
             headers["OpenAI-Organization"] = api_org
         r = self.retry(
-            lambda: requests.get(
-                "https://api.openai.com/v1/models",
-                headers=headers,
-            ),
+            call_method=requests.get,
+            url="https://api.openai.com/v1/models",
+            headers=headers,
+            timeout=self._max_request_time,
         )
         if r.status_code == 422:
```

Comment:
Could we add an enum for these error codes? See [here](https://github.com/explosion/spacy-llm/pull/110/files#diff-1c49ce9ad523ef937d590bdf0fc1f62143ef7b0fea500f28d34c23d190d12e38L48).

Reply:
Done in https://github.com/explosion/spacy-llm/pull/110/commits/d28a5da6eb9c4796b90136b3bc35b68f5b70d115.